src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -466,19 +484,19 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
   }
 };
 
-static int open_device(const char* path, bool try_load_module = false)
+static int open_device(const char* path, Config *cfg = NULL, bool try_load_module = false)
```

Comment:
A short answer: with modern C++ use `nullptr`. For more details see e.g [1].

There is no much sense in changing the current code just to convert `NULL` to `nullptr`, but in a new code use `nullptr`.

[1] http://en.cppreference.com/w/cpp/language/nullptr